github/orgs_outside_collaborators.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import "fmt"
+
+// ListOutsideCollaboratorsOptions specifies optional parameters to the
+// OrganizationsService.ListOutsideCollaborators method.
+type ListOutsideCollaboratorsOptions struct {
+	// Filter outside collaborators returned in the list. Possible values are:
+	// 2fa_disabled, all.  Default is "all".
+	Filter string `url:"filter,omitempty"`
+
+	ListOptions
+}
+
+// ListOutsideCollaborators lists outside collaborators of organization's repositories.
+// This will only work if the authenticated
+// user is an owner of the organization.
+//
+// Warning: The API may change without advance notice during the preview period.
+// Preview features are not supported for production use.
```

Comment:
We don't usually include this warning in all other similar situations. Should we start doing it here?

It'll create more work and might get out of date. We have the `// TODO: remove custom Accept header when this API fully launches.` comments in code, and GitHub API itself is a good source for knowing which endpoints are stable or not.

I'm not against it, just a thought, because this is the first comment of its kind.